setup.cfg

```
@@ -0,0 +1,23 @@
+[metadata]
+name = ADAM
+description = Automatic Differentiation for rigid body AlgorithMs
+author = "Giuseppe L'Erario"
+author_email = gl.giuseppelerario@gmail.com
+license_file = LICENSE
+
+[options]
+packages = find:
+python_requires = >=3.7
+install_requires =
+        numpy
+        scipy
+        casadi
+        matplotlib
+        prettytable
+        urdf_parser_py
+
+[options.extras_require]
+test =
+    pytest
+    idyntree
+    gym-ignition
```

```suggestion
    gym-ignition-models
```

Before was working because gym-ignition depends on gym-ignition-models.

---

Ops :) Fixed in 18d8576.